lms/djangoapps/commerce/views.py

```diff
@@ -137,4 +139,23 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
 def checkout_cancel(_request):
     """ Checkout/payment cancellation view. """
     context = {'payment_support_email': microsite.get_value('payment_support_email', settings.PAYMENT_SUPPORT_EMAIL)}
-    return render_to_response("commerce/checkout_cancel.html", context)
+    return mako_render_to_response("commerce/checkout_cancel.html", context)
+
+
+@csrf_exempt
+def checkout_receipt(_request):
+    """ Receipt view. """
+    context = {'platform_name': settings.PLATFORM_NAME}
```

Comment:
I suspect this should be microsites-aware as well, e.g. `microsite.get_value('platform_name', settings.PLATFORM_NAME)`